platforms/documentation/docs/src/samples/credentials-handling/publishing-credentials/groovy/maven-repository-stub/src/main/groovy/maven-repository-stub.gradle

```diff
@@ -2,7 +2,7 @@ tasks.withType(PublishToMavenRepository).configureEach {
     notCompatibleWithConfigurationCache("Configures repository at execution time")
     doFirst {
         def address = com.example.MavenRepositoryStub.start()
-        getRepository().setUrl(address)
+        getRepository().get().setUrl(address)
```

Comment:
This sample seems really incorrect. We should not be mutating the URL of the repository on the task itself.